src/core/trulens/core/schema/select.py

```diff
@@ -0,0 +1,146 @@
+"""Serializable selector-related classes."""
```

Comment:
Moved this out in an effort to get as many top-level imports as possible and this would cause a circular import in `feedback.py`.

Reply:
Thanks for doing this.